bin/docker_push

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -evuo pipefail
+
+VERSION=$(python setup.py --version)
+DOCKER_IMAGE="fiaas/skipper:${VERSION/+/-}"
+
+echo "${DOCKER_PASSWORD}" | docker login -u "${DOCKER_USERNAME}" --password-stdin
+
+docker tag "fiaas/skipper:latest" "${DOCKER_IMAGE}"
+
+if [[ "${CI}" == "true" ]]; then
```

Comment:
This variable is always true. You are checking the value for cases when you run it locally?

Reply:
Yes. It made more sense to test the scripts this way.